### DISCUSSION TITLE
Is it possible to restart ESPAltherma remotely?

### DISCUSSION THREAD

**Discussion by mydliar** [raomin/ESPAltherma#314] [Q&A]
Hi,
Is it possible to restart ESPAltherma remotely? I am away and can't turn the device (M5Stack M5StickC PLUS ESP32-PICO Mini) off/on manually. It stopped providing MQQT feed two days ago, not sure where the problem is, but the device itself seems to be still online (can ping it).
Thanks,

Martin

**Comment by bugac** [comment]
hi, im using homeassistant+hacs addon (screen below)
![image](https://github.com/raomin/ESPAltherma/assets/404278/9977e20b-ce30-4910-9611-1debdee364ad)

**Comment by riksmithuk** [comment]
Did you find a way to remotely restart ESPAltherma - i am now also in a similar position.
